EventFilter/CTPPSRawToDigi/interface/DiamondVFATFrame.h

```diff
@@ -28,14 +31,14 @@ class DiamondVFATFrame : public VFATFrame
     /// get timing infromation
     uint32_t getLeadingEdgeTime() const
     {
-      uint32_t time = ((data[5]&0x1f)<<16)+data[6];
+      uint32_t time = ((data[7]&0x1f)<<16)+data[8];
```

Comment:
from the PR description it sounds like this swap has always been there and that all past data will need this fix.